docs/Sources/tuist/tuist.docc/Articles/Tuist/Extensions/generation-environment.md

```diff
@@ -0,0 +1,49 @@
+# Generation-time configuration
+
+Learn how to leverage environment variables that can be read from the manifest files.
+
+There are certain scenarios where you might need to customize the projects at generation time. For example, if you are building a white-label app, you might want to use the same project structure, but adjust some attributes in a per-app basis. For example, the name of the app.
```

Comment:
Instead of saying "you might need to customize at generation time", which one might think that's already possible right in the manifest files, I'd say that "you might need want to control the shape of the generated projects when invoking Tuist".